dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1283,7 +1282,7 @@ async fn lookup_service_info(
     blueprint: &Blueprint,
 ) -> anyhow::Result<Option<ServiceInfo>> {
     let Some(zone_config) = blueprint
-        .all_omicron_zones(BlueprintZoneFilter::All)
+        .all_omicron_zones(|_disposition| true)
```

Comment:
Would you consider adding `BlueprintZoneDisposition::any` for this?

Reply:
Ooh, I like this!

Reply:
I thought about it and thought it was kind of weird; to be clear that method implementation is

```rust
impl BlueprintZoneDisposition {
    fn any(self) -> { true }
}
```

which seems pretty strange in isolation. No argument about it making these call sites nicer, though! I guess it just needs an adequate doc comment?

Reply:
added in [61d3edd](https://github.com/oxidecomputer/omicron/pull/7561/commits/61d3edd2a941f3bac1f96142276595ad71daad88)